stdlib.sh

```diff
@@ -346,7 +346,7 @@ source_env() {
   pushd "$(pwd 2>/dev/null)" >/dev/null || return 1
   pushd "$rcpath_dir" >/dev/null || return 1
   if [[ -f ./$rcpath_base ]]; then
-    log_status "loading $rcfile"
+    log_status "loading $(expand_path "$rcpath")"
```

Comment:
Should the `expand_path` be amended to show `~` instead of user home path?
Kind of:
```suggestion
    log_status "loading $(expand_path "$rcpath" | sed 's|^'${HOME:=none}'|~|')"
```

Reply:
@yermulnik thanks for catching the regression! `user_rel_path` did it eventually.